pkg/infinispan/client/client.go

```diff
@@ -16,6 +16,7 @@ To prevent duplicated code, newer api implementation packages should use composi
 function as expected on newer server versions.
 
 For example, if v14 can reuse a new `api.Cache` implementation but requires a new `api.Caches` implementation:
+
```

Comment:
Why is this and [pkg/apis/integreatly/v1alpha1/grafanadashboard_types.go](https://github.com/infinispan/infinispan-operator/pull/1802/files#diff-73f5e8039ea36620af1fb7969676000b6c82e3bfe2c3dc70c0220e2870893d8e) change required?

Reply:
probably some wrong setting in my Go installation. When I run `make manager`, the format is changed in a couple of files and I probably miss tho revert those 2.